back/boxtribute_server/models/definitions/history.py

```diff
@@ -25,7 +25,17 @@ class DbChangeHistory(db.Model):
 
     class Meta:
         table_name = "history"
-        indexes = (
-            (("record_id", "change_date"), False),
-            (("table_name", "record_id", "change_date"), False),
-        )
+
+
+DbChangeHistory.add_index(
+    DbChangeHistory.record_id,
+    DbChangeHistory.change_date,
+    name="recordid_changedate_composite_index",
+)
+DbChangeHistory.add_index(
+    DbChangeHistory.table_name,
+    DbChangeHistory.record_id,
+    DbChangeHistory.change_date,
+    name="tablename_recordid_changedate_composite_index",
+)
+DbChangeHistory.add_index(DbChangeHistory.user, name="user_id")
```

Comment:
@vahidbazzaz 
The lines above are the index definitions such that peewee is aware of table indexes. In the back-end testing environment the database tables are set up before every test, and these definitions help to have the same DB structure in testing as in production.
The peewee index definitions don't have an influence in production (they will only once we start to perform DB migrations via Python instead of Sphinx), since the indexes are defined in the MySQL DB.

I know that you recommended additional indexes. I've copied these recommendations into #1203 but also made a remark that the query is quite fast already. Hence I didn't write a Sphinx migration for adding another index to the history table.